sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1659,6 +1667,8 @@ static void *_gnrc_netif_thread(void *args)
     dev->context = netif;
     /* initialize low-level driver */
     res = dev->driver->init(dev);
```

Comment:
we could now also use this return value as the return value for `gnrc_netif_create()` which currently will always return 0.

Reply:
How would you hand that over to the thread-creating thread?

Reply:
either by using a global variable or by passing a

```C
struct {
    gnrc_netif_t *netif;
    int result;
}
```

in `args`

Reply:
As this would also provide the potential to free the `init_done` memory space, I think I would prefer the latter method.

Reply:
Ok